addons/skin.estuary/xml/DialogSeekBar.xml

```diff
@@ -31,7 +31,7 @@
 				</control>
 				<control type="label">
 					<top>110</top>
-					<left>20</left>
+					<left>250</left>
```

Comment:
the 'ends at' label starts at left 20 and has a width of 400, so it could potentially overlap with this one.
so we're going to need a bit of tweaking..

please set left to 270 here